### PR TITLE
Sql azure task publish profile fix for default working directory

### PR DESCRIPTION
**Pull request by mvvsubbu**

### CODE REVIEW

File: Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1 [hunk 1]
```diff
@@ -61,8 +61,8 @@ Write-Host "packageFile= $DacpacFilePath"
 #Ensure that a single package (.dacpac) file is found
 ThrowIfMultipleFilesOrNoFilePresent -files $DacpacFilePath -pattern $DacpacFile
 
-$PublishProfilePath=""
-if( !([string]::IsNullOrWhitespace($PublishProfile)) )
+$PublishProfilePath = ""
+if( [string]::IsNullOrWhitespace($PublishProfile) -eq $false -and $PublishProfile -ne $env:SYSTEM_DEFAULTWORKINGDIRECTORY )
```
Comment (kmkumaran) [inline Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1]: I remember DeployRG doing one more check comparing against $env:SYSTEM_DEFAULTWORKINGDIRECTORY/ with a / in end.

**Comment by kmkumaran** [comment]
Looks fine.

**Comment by anaggar** [comment]
Fine

File: Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1 [hunk 2]
```diff
@@ -61,8 +61,8 @@ Write-Host "packageFile= $DacpacFilePath"
 #Ensure that a single package (.dacpac) file is found
 ThrowIfMultipleFilesOrNoFilePresent -files $DacpacFilePath -pattern $DacpacFile
 
-$PublishProfilePath=""
-if( !([string]::IsNullOrWhitespace($PublishProfile)) )
+$PublishProfilePath = ""
+if( [string]::IsNullOrWhitespace($PublishProfile) -eq $false -and $PublishProfile -ne $env:SYSTEM_DEFAULTWORKINGDIRECTORY -and $PublishProfile -ne [String]::Concat($env:SYSTEM_DEFAULTWORKINGDIRECTORY, "\"))
```
Comment (anaggar) [inline Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1]: Can we make a function for checking this one ?

**Comment by Ajay-MS** [comment]
Looks fine

**Pull request merged**